### PR TITLE
fix(selenium): add backwards compatible way of settings various timeouts when using selenium 3

### PR DESCRIPTION
**Pull request by Zidious**
This PR adds adds a selenium 3 compatible fallback when we attempt to set the various timeouts. 

- Added a regression test to catch this
- Found two unrelated issues (tagged in this commit that will require follow up shortly after)

QA Notes: Using Selenium 3 it should analyze correctly there should also be no change to the existing behaviour when using selenium 4. 

Closes: https://github.com/dequelabs/axe-core-maven-html/issues/479

### CODE REVIEW

File: selenium/pom.xml [hunk 1]
```diff
@@ -1,215 +1,224 @@
...
+            <scope>test</scope>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/org.mockito/mockito-core -->
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <!-- versions >4 require Java 11 -->
+            <version>4.11.0</version>
+            <scope>test</scope>
+        </dependency>
```
Comment (Zidious) [inline selenium/pom.xml:216]: Mostly formatting changes - this test dependency is the new addition.

**Review by stephenmathieson — APPROVED** [review]
LGTM

We should really consider dropping support for Selenium 3, and do so quickly. Is this on the roadmap?

**Comment by Zidious** [comment]
> LGTM
> 
> We should really consider dropping support for Selenium 3, and do so quickly. Is this on the roadmap?

I believe this has been in discussion for the 5 release. As updating to the the later versions of Selenium 4 does require Java 11. So we'd need to think carefully whether we want to continue to support selenium 4.X on Java 8 or move to selenium >4 on Java 11 or both

**Comment by stephenmathieson** [comment]
Is there an issue for tracking removing Selenium 3 support?

**Comment by Zidious** [comment]
> Is there an issue for tracking removing Selenium 3 support?

- https://github.com/dequelabs/axe-core-maven-html/issues/378
- https://github.com/dequelabs/axe-core-maven-html/issues/411

**Pull request merged**